### ISSUE TITLE
[BUG] About menu 

### ISSUE THREAD

**Issue by drc5521** [kiwibrowser/src.next#174]
When I open the Settings, Downloads, or Bookmarks, and then drag the browser to the background, and then open it, it will reload.The mode will also switch back and forth.
See the video for details.
https://user-images.githubusercontent.com/55195642/126870526-0c07210d-4a54-47bb-b48d-b3a491d7c588.mp4

**Comment by karl-pae** [comment]
I think tap on icon, re launches the app. If you switch using app manager it doesn't reload. This is expected behaviour I guess. Same happens with Google Chrome. 

@kiwibrowser What do you say?

**Comment by KaKi87** [comment]
Can you reproduce this bug on the latest Kiwi Browser from Play Store ? Because I can't

**Comment by drc5521** [comment]
> Can you reproduce this bug on the latest Kiwi Browser from Play Store ? Because I can't

Downloading and installing in the play store is the same result.The same is true for Generation 1131271226. It happened on Android 10 and Android 7.

**Comment by karl-pae** [comment]
Can you try this again with latest release. This doesn't happen with me anymore.

**Comment by drc5521** [comment]
> Can you try this again with latest release. This doesn't happen with me anymore.

Yes, the problem still exists. The pattern will not change, but the first problem still exists.

**Comment by d3ward** [comment]
That bug seems to be related to Android version and Chromium code . 
Tested with Kiwi and also with Chrome and any other chromium browser this bug exist, so technically not Kiwi related bug .
Settings page is created like at runtime with an activity , seems like when you/we put the app in background the activity is closed to optimize battery .

Also something very interesting behaviour . Go to settings and put on background .
If you open from the app switcher of Android the setting will be still open , if you open the app by shortcut from launcher settings will close
